shared/utils/relabel-ids.py

```diff
@@ -206,13 +207,16 @@ def drop_oval_checks_extending_non_existing_checks(ovaltree, indexed_oval_defs):
 
             if referreddefinition is None:
                 # There is no oval satisfying the extend_definition referal
-                defstoremove.append((definition, extdefinitionref))
+                incomplete = True
+                sys.stderr.write("WARNING: OVAL check '%s' extends non-existing '%s' "
+                "OVAL definition, removing check from OVAL definitions.\n"
+                % (definition.get("id"), extdefinitionref))
```

Comment:
I think that printing it for every occurrence is more useful, imagine a definition that extends several others but none is found, if we print just the first, you'll have to fix and rebuild several times until it works.